source/adios2/ADIOSConfig.h.in

```diff
@@ -42,4 +42,7 @@
 /* CMake Option: ADIOS_USE_DataMan=TRUE */
 #cmakedefine ADIOS2_HAVE_DATAMAN
 
+/* CMake Option: ADIOS_USE_Python=TRUE */
+#cmakedefine ADIOS2_HAVE_Python
```

Comment:
I was thinking if we have two python versions present. That's how I tested Python 2.7.x and 3.5 in the same system. I am assuming most users will have Python 2.7.x